### PR TITLE
test: disable object number e2e tests with hardcoded colors

### PR DESCRIPTION
**Pull request by droshev**
disable object number e2e tests with hardcoded colors

### CODE REVIEW

File: e2e/wdio/core/tests/object-number.e2e-spec.ts [hunk 1]
```diff
@@ -54,23 +54,23 @@ describe('object number test suite', function () {
     });
 
     describe('basic object number examples', function () {
-        it('should check colors and values', () => {
+        xit('should check colors and values', () => {
```
Comment (InnaAtanasova) [inline e2e/wdio/core/tests/object-number.e2e-spec.ts:57]: wouldn't it be easier if we disable the whole block starting on line 25?

Comment (droshev) [reply e2e/wdio/core/tests/object-number.e2e-spec.ts:57]: there are not only color checks

**Review by InnaAtanasova — APPROVED** [review]

**Comment by nikvalor** [comment]
Project: null Platform B
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform A 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

**Pull request merged**